docs/godmode.md

```diff
@@ -14,7 +14,7 @@ The essential idea is to reverse the process of testing. We use the _actual_ sou
 
 GodMode can be activated in two different ways.
 
-1. If we want to execute arbitrary tests using GodMode, then update `BaseUiTestCase` class (or `EmailGeneratorTest` class if testing email content) and set `isGodModeEnabled = true` at the top of the class implementation. Now all test runs would have GodMode enabled. Please remember to set it back to false when done.
+1. If we want to execute arbitrary tests using GodMode, then update `test.properties` to include the line `test.godmode.isenabled=true`. Now all test runs would have GodMode enabled. Please remember to set it back to `false` when done.
```

Comment:
I guess I was thinking more for the current developers who don't currently have that line in their `test.properties`. Same motivation for using
```
Boolean.parseBoolean(prop.getProperty("test.godmode.isenabled", "false"));
```
instead of just
```
Boolean.parseBoolean(prop.getProperty("test.godmode.isenabled"));
```
which will throw an exception for existing `test.properties` files which don't have that property defined. Should I change it to the latter to force everyone to add the line `test.godmode.isenabled=false` to their `test.properties`? Or just make the assumption in the documentation but not in the code?